tests/consistent.rs

```diff
@@ -0,0 +1,213 @@
+extern crate regex;
+extern crate quickcheck;
```

Comment:
I generally try to keep `extern crate` statements in the crate root, even if this means repeating some `cfg`s.